src/pages/dataTableUtilities/getIndicatorTableData.js

```diff
@@ -202,10 +95,11 @@ const mapIndicatorTableColumn = column => {
 /**
  * Map indicator columns to data table column objects.
  * @param {Array.<IndicatorAttribute>} indicatorColumns
+ * @oaram {bool} isEditable
  * @return {Array.<object>}
  */
-const mapIndicatorTableColumns = indicatorColumns => {
-  const valueColumns = indicatorColumns.map(mapIndicatorTableColumn);
+const mapIndicatorTableColumns = (indicatorColumns, isEditable) => {
```

Comment:
as above the `get` and `map` pretty excessive

Reply:
ac0b5f3465f6e3ff6af6b5064ac36f2c7185006f.